math/src/field/element.rs

```diff
@@ -39,7 +40,7 @@ impl<F: IsField> FieldElement<F> {
         for i in 1..count {
             prod_prefix.push(&prod_prefix[i - 1] * &numbers[i]);
         }
-        let mut bi_inv = prod_prefix[count - 1].inv();
+        let mut bi_inv = prod_prefix[count - 1].inv().unwrap();
```

Comment:
Same issue. We could settle for postponing in this case if needed though, to keep the splash damage more contained.